pkg/core/tokens/issuer_test.go

```diff
@@ -144,7 +144,7 @@ var _ = Describe("Token issuer", func() {
 			err = validator.ParseWithValidation(ctx, token, id)
 
 			// then
-			Expect(err.Error()).To(ContainSubstring("could not parse token: token is expired"))
+			Expect(err).To(HaveOccurred())
```

Comment:
There's still value in checking the errors no?

Reply:
I mean the content of the error message

Reply:
I'm not sure why. We don't check the message in the program, the test of the error isn't part of the interface of this module. This is something that can only fail in this test and it's fragile.